assets/app/views/browse/pod.html

```diff
@@ -95,7 +95,7 @@
                         context="projectContext"
                         options="logOptions"
                         status="pod.status.phase"
```

Comment:
While you're in the code, can you remove pod status to fix #7414? We might want to show container state at some point later, but that's the quick fix.